.circleci/config.yml

```diff
@@ -47,25 +55,18 @@ jobs:
           name: Which bundler?
           command: bundle -v
 
-      - restore_cache:
-          keys:
-            - suri-rails-dependencies-{{ checksum "Gemfile.lock" }}
-            # fallback to using the latest cache if no exact match is found
-            - suri-rails-dependencies-
-
```

Comment:
we don't want to cache because ...

Reply:
@ndushay Good question! This caching is messed up in a few ways. First, it uses different keys for restore and save (dor-services vs. suri-rails). Second, I don't believe the cache is persisted across runs. So, basically, I think this is copypasta from some of our other repos where caching *has* been useful (e.g., in our GoLang and React work), and I'm removing it from Ruby/Rails projects whenever I see it.